src/main/java/org/opensearch/security/auditlog/impl/AuditCategory.java

```diff
@@ -33,13 +33,8 @@ public enum AuditCategory {
     COMPLIANCE_INTERNAL_CONFIG_WRITE;
 
     public static Set<AuditCategory> parse(final Collection<String> categories) {
-        if (categories.isEmpty())
-            return Collections.emptySet();
+        if (categories.isEmpty()) return Collections.emptySet();
```

Comment:
🫠 I would prefer we always have curly brackets on if statements - can this be tweaked?

Reply:
Yeah, I can look into modifying the formatting file to include that!

Reply:
Let me know if you think of any other style guides you would like.